php/libraries/Database.class.inc

```diff
@@ -760,15 +760,15 @@ class Database
     }
 
     /**
-     * Runs a query as a prepared statement and returns the values of the first
-     * column in the select statement. If multiple columns are given, only the first
-     * will be returned.
+     * Runs a query as a prepared statement and returns the values of the
+     * column given in the select statement. If multiple columns are given, an error
+     * is thrown.
```

Comment:
@zaliqarosli you should also add  the @throws tag with proper comment

Reply:
the @throws tag has been added on the major branch already... potential for conflicts

https://github.com/aces/Loris/blob/major/php/libraries/Database.class.inc#L846

Reply:
@zaliqarosli @driusan I just realized the tag is being added in this pr https://github.com/aces/Loris/pull/3511/files#diff-c1d4495bb4192e908841940828b887a4L773

you can forget about it here... sorry. I'm cancelling my review